Packages/com.unity.inputsystem/ValidationExceptions.json

```diff
@@ -1,4 +1,9 @@
 {
-    "ErrorExceptions": [],
+    "ErrorExceptions": [
```

Comment:
Any reason to have this error exception? I

Reply:
removed this again since we didn't have this in the last few releases either